tensorboard/webapp/metrics/actions/index.ts

```diff
@@ -211,6 +213,22 @@ export const dataTableColumnDrag = createAction(
   }>()
 );
 
+export const dataTableColumnEdited = createAction(
+  '[Metrics] Data table columns edited in edit menu',
+  props<{
+    fobState: FobState;
+    newHeaders: ColumnHeader[];
```

Comment:
Maybe just "headers" instead of "newHeaders"?

Reply:
Done.